testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -29,7 +29,9 @@
 import io.trino.spi.connector.ConnectorFactory;
```

Comment:
add a test where column has multiple referenced columns (source columns)

Reply:
I guess it is already added.